openmetadata-ui/src/main/resources/ui/src/pages/DatasetDetailsPage/DatasetDetailsPage.component.tsx

```diff
@@ -226,10 +233,10 @@ const DatasetDetailsPage: FunctionComponent = () => {
         setEntityLineage(res.data);
       })
       .catch((err: AxiosError) => {
-        showToast({
-          variant: 'error',
-          body: err.message ?? 'Error while fetching lineage data.',
-        });
+        const msg =
```

Comment:
```suggestion
        const message =
```
Please make this change in other places too.